### PR TITLE
Refactor addErrorLog Parameters + Uniform Error Variable Names

### LINKED ISSUE #204: Remove All Console.log() Statements
This a good link on reference: [Error Handling ](https://reactjs.org/blog/2017/07/26/error-handling-in-react-16.html).

### PR DESCRIPTION
**Pull request by seemavora**
Makes sure that the errorDescription actually calls 'err'.  :)  

Resolves #204

### CODE REVIEW

File: api/routes/Event.js [hunk 1]
```diff
@@ -14,11 +14,21 @@ const {
...
-    .then(items => res.status(OK).send(items));
+    .then(items => res.status(OK).send(items))
+    .catch(err => {
+      const info = {
+        errorTime: new Date(),
+        apiEndpoint: 'Event/getEvents',
+        errorDescription: err
+      };
+      addErrorLog(info);
+      res.status(BAD_REQUEST).send({ err, message: 'getting event failed' });
```
Comment (jerrylee17) [inline api/routes/Event.js:30]: `message: Getting event failed`

Comment (seemavora) [reply api/routes/Event.js]: done :)

File: api/routes/3DPrintingForm.js [hunk 2]
```diff
@@ -45,7 +46,13 @@ router.post('/GetForm', (req, res) => {
 
   PrintingForm3D.find(obj, (error, forms) => {
     if (error) {
-      logger.log(`3DPrinting /GetForm error: ${error}`);
+      const info = {
+        userEmail: req.body.email,
+        errorTime: new Date(),
+        apiEndpoint: '3DPrintingForm/GetForm',
+        errorDescription: err
```
Comment (evanugarte) [inline api/routes/3DPrintingForm.js:53]: you checked that this was a string?

File: api/routes/officerManager.js [hunk 3]
```diff
@@ -53,7 +54,13 @@ router.post('/GetForm', (req, res) => {
 
   Manager.find(obj, (error, forms) => {
     if (error) {
-      logger.log(`Officer Manager /GetForm error: ${error}`);
+      const info = {
+        userEmail:req.body.email,
+        errorTime: new Date(),
+        apiEndpoint: 'officerManager/GetForm',
+        errorDescription: err
```
Comment (evanugarte) [inline api/routes/officerManager.js:61]: `err` is undefined.

File: api/routes/officerManager.js [hunk 4]
```diff
@@ -71,7 +78,13 @@ router.post('/delete', (req, res) => {
   }
   Manager.deleteOne({ email: req.body.email }, function(error, form) {
     if (error) {
-      logger.log(`Officer Manager /Delete3DForm error: ${error}`);
+      const info = {
+        userEmail: req.body.email,
+        errorTime: new Date(),
+        apiEndpoint: 'officerManager/delete',
+        errorDescription: err
```
Comment (evanugarte) [inline api/routes/officerManager.js:85]: again, `err` is undefined

File: api/routes/officerManager.js [hunk 5]
```diff
@@ -101,7 +114,13 @@ router.post('/edit', (req, res) => {
 
   Manager.updateOne(query, { ...form }, function(error, result) {
     if (error) {
-      logger.log(error);
+      const info = {
+        userEmail:req.body.email,
+        errorTime: new Date(),
+        apiEndpoint: 'officerManager/edit',
+        errorDescription: err
```
Comment (evanugarte) [inline api/routes/officerManager.js:121]: fix these lines and ill review the rest

File: api/routes/3DPrintingForm.js (hunk 2 above)
Comment (jerrylee17) [reply api/routes/3DPrintingForm.js:53]: [https://docs.mongodb.com/stitch/mongodb/actions/collection.find/](Yes)

**Comment by jerrylee17** [comment]
Squash commits before pushing

**Comment by ThaiQ** [comment]
@seemavora Travis FAIL!

**Comment by ThaiQ** [comment]
How to read Travis logs:
1. Details
image

2. Build failed
image

3. Click the failing node
image

4. Error logs
image

File: api/routes/InventoryItem.js [hunk 6]
```diff
@@ -44,15 +45,15 @@ router.post('/editItem', (req, res) => {
             .status(OK)
             .json({ ret, item: 'Inventory item updated successfully' });
         })
-        .catch(err => {
+        .catch(error => {
```
Comment (evanugarte) [inline api/routes/InventoryItem.js:48]: why are we changing every `err` to `error`?

File: api/routes/InventoryItem.js [hunk 7]
```diff
@@ -44,15 +45,15 @@ router.post('/editItem', (req, res) => {
...
-            err,
+            error,
             message: 'Inventory item was not updated'
           });
         });
     })
-    .catch(err => {
-      res.status(NOT_FOUND).send({ err, message: 'item not found' });
+    .catch(error => {
+      res.status(NOT_FOUND).send({ error, message: 'item not found' });
```
Comment (evanugarte) [inline api/routes/InventoryItem.js:56]: i dont get why this line is changed

File: api/routes/user.js [hunk 8]
```diff
@@ -22,6 +22,9 @@ const {
   CONFLICT
 } = require('../constants').STATUS_CODES;
 const membershipState = require('../constants').MEMBERSHIP_STATE;
+const addErrorLog = require ('../util/errorLog');
+
+
```
Comment (evanugarte) [inline api/routes/user.js:27]: remove this blank line and the one above it

File: api/util/errorLog.js [hunk 9]
```diff
@@ -9,13 +9,13 @@ const ErrorLog = require('../models/ErrorLog');
  * @param {string} req.body.apiEndpoint - The location of the error
  * @param {string} req.body.errordescription - The description of the error
  */
-async function addErrorLog(userEmail, errorTime, apiEndpoint, errorDescription){
+async function addErrorLog(req){
```
Comment (evanugarte) [inline api/util/errorLog.js:12]: lets call this something better like

```suggestion
async function addErrorLog(errorToAdd){
```

File: api/util/errorLog.js [hunk 10]
```diff
@@ -9,13 +9,13 @@ const ErrorLog = require('../models/ErrorLog');
  * @param {string} req.body.apiEndpoint - The location of the error
```
Comment (evanugarte) [inline api/util/errorLog.js:9]: change the variable names of these comments, e.g.
```suggestion
 * @param {string} errorToAdd.apiEndpoint - The location of the error
```

File: src/Pages/3DPrintingConsole/3DConsole.js [hunk 11]
```diff
@@ -88,7 +88,7 @@ export default class PrintConsole3D extends React.Component {
     }
   }
 
-  // simply filter array by name
+  // simply filter array by name!
```
Comment (evanugarte) [inline src/Pages/3DPrintingConsole/3DConsole.js:91]: remove this line all together

**Comment by evanugarte** [comment]
@seemavora also rename this pr

File: src/Pages/3DPrintingConsole/3DConsole.js (hunk 11 above)
Comment (seemavora) [reply src/Pages/3DPrintingConsole/3DConsole.js:91]: oops we were using this line to test grpc issue, and left it

File: api/routes/ErrorLog.js [hunk 12]
```diff
@@ -4,9 +4,8 @@ const ErrorLog = require('../models/ErrorLog');
 const { OK, BAD_REQUEST } = require('../constants').STATUS_CODES;
 const { addErrorLog } = require('../util/errorLog');
 
-router.post('/addErrorLog', async (req, res) => {
-  if (!await addErrorLog(req.body.userEmail, req.body.errorTime,
-    req.body.apiEndpoint, req.body.errorDescription)) {
+router.post('/addErrorLog', async (errorToAdd, res) => {
```
Comment (evanugarte) [inline api/routes/ErrorLog.js:7]: just change the addErrorLog function param name, no need for anything else
```suggestion
router.post('/addErrorLog', async (req, res) => {
```

**Pull request merged**